docs/data/_index.md

```diff
@@ -19,23 +19,33 @@ menuindent: true
 # SME: Alexa Greenberg
 ---
 
-Viam's built-in data management service allows you to capture, sync, [view](view/), [label](dataset/), and [export](export/) data from your machines, as well as use [Machine Learning](/ml/) features to enhance your machine's capabilities.
-The service is designed for flexibility and efficiency while preventing data loss, data duplication, and other data management issues.
+The data management service is a robust solution for smart machine data handling, [data capture](/data/capture/), and [cloud synchronization](/data/cloud-sync/).
+Using the data management service, you can collect data from different parts of a robot, IoT device, or any other machine.
+Then, you can configure which data to sync to the cloud, and view, sort, or manage it directly in the cloud without needing to manually gather data from each machine.
+The data management service also keeps your data secure during transfer.
+
+## Overview
 
 ![Data is captured on the machine, uploaded to the cloud, and then deleted off local storage.](/data/data_management.png)
 
-The service has two parts: [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/).
+Viam's data management features can be broken down into on-device and cloud data capabilities:
 
-Once you have captured and synced data, you can:
+- **On-Device Data Management:** Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+  - **[Data Capture](/data/capture/):** Configurable for various robot components like cameras, sensors, and motors; set capture frequency, with automatic resumption after robot restarts.
+  - **[Cloud Sync](/data/cloud-sync/):** Securely transfers data to cloud at user-defined frequency, resilient to interruptions, and deletes local data post-sync for space management.
```

Comment:
I think this looks a bit cramped. How would you feel about cards instead? (For this and the cloud data management section)
```suggestion
### On-Device Data Management

Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.

{{< cards >}}
{{% manualcard link="/data/capture/" %}}

#### Data Capture

Configurable for various robot components like cameras, sensors, and motors; set capture frequency, with automatic resumption after robot restarts.

{{% /manualcard %}}
{{% manualcard link="/data/cloud-sync/" %}}

#### Cloud Sync

Securely transfers data to cloud at user-defined frequency, resilient to interruptions, and deletes local data post-sync for space management.

{{% /manualcard %}}
{{< /cards >}}
```

Reply:
Great suggestion! Will implement.